Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/RequirementData.xml

```diff
@@ -2897,7 +2897,7 @@
     </CRequirement>
     <CRequirement id="AP_HaveDestroyerChargingBeam">
         <EditorCategories value="Race:Protoss,TechType:Ability"/>
-        <NodeArray index="Show" Link="AP_CountUpgradeDestroyerChargingBeamCompleteOnly"/>
+        <NodeArray index="Use" Link="AP_CountUpgradeDestroyerChargingBeamCompleteOnly"/>

```

Comment:
This change isn't required as vanilla destroyers are without this trait

Reply:
I argue it is necessary for consistency with the other war council upgrades.